frontend/src/pages/exploreApplication/ExploreApplications.tsx

```diff
@@ -19,18 +55,30 @@ const ExploreApplications: React.FC = () => {
       loadError={loadError}
     >
       {!isEmpty ? (
-        <Gallery className="odh-explore-apps__gallery" hasGutter>
-          {components
-            .sort((a, b) => a.spec.displayName.localeCompare(b.spec.displayName))
-            .map((c) => (
-              <OdhExploreCard
-                key={c.metadata.name}
-                odhApp={c}
-                isSelected={selectedComponent === c.metadata.name}
-                onSelect={() => setSelectedComponent(c.metadata.name)}
-              />
-            ))}
-        </Gallery>
+        <Drawer isExpanded={!!selectedComponent} isInline>
+          <DrawerContent
+            panelContent={
+              <GetStartedPanel onClose={() => updateSelection()} selectedApp={selectedComponent} />
```

Comment:
Should you be able to close the panel by clicking the card again and using the ESC key?

Reply:
Don't think we need to trap the ESC key.